### DISCUSSION TITLE
xformers are missing

### DISCUSSION THREAD

**Discussion by Aribk7** [TheLastBen/fast-stable-diffusion#911] [General]
any timeline on when this can get fixed sending out a email to 130k people with this colab.

**Comment by jonas-becker** [comment]
Related to this issue:

`/usr/local/lib/python3.8/dist-packages/xformers/_C.so: undefined symbol: _ZNK3c104impl13OperatorEntry20reportSignatureErrorENS0_12CppSignatureE
WARNING:xformers:WARNING: /usr/local/lib/python3.8/dist-packages/xformers/_C.so: undefined symbol: _ZNK3c104impl13OperatorEntry20reportSignatureErrorENS0_12CppSignatureE
Need to compile C++ extensions to get sparse attention support. Please run python setup.py build develop`

It was working yesterday, but today I am getting this warning. May also be related to #387. Tried it on multiple GPU in colab and deleted the "sd" folder from my drive multiple times.

**Reply by Maxadox** [reply]
same here

**Comment by TheLastBen** [comment]
fixed https://github.com/TheLastBen/fast-stable-diffusion/commit/3cf2052a3d3f837801b5d77cb136aa276653a261

**Comment by Aribk7** [comment]
keep getting this error 

  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 11, in <module>
    from modules.call_queue import wrap_queued_call, queue_lock, wrap_gradio_gpu_call
ModuleNotFoundError: No module named 'modules.call_queu

**Reply by Aribk7** [reply]
I dmed u on twitter

**Reply by TheLastBen** [reply]
are you using the links from the main page ? don't save the colabs to your gdrive, the become outdated fast

**Reply by DrakenZA** [reply]
Seems like the A100 pre-compile is also broken.

What is wrong with this shitty xformers that we need to re-compile it every week.

**Reply by TheLastBen** [reply]
google colab changed something, not xformers

**Comment by TheLastBen** [comment]
procedure to get the A100 pre-compiled libs, run this in a new cell : 

!pip install ninja
!pip install -v -U git+https://github.com/facebookresearch/xformers.git@main#egg=xformers
%cd /content
!zip -r A100 /usr/local/lib/python3.8/dist-packages/xformers
!cp A100.zip /content/gdrive/MyDrive

after that share the file in you gdrive and post the link here.

**Reply by goblincore** [reply]
Ran the above, this fixed my issue 👍 
https://drive.google.com/file/d/1-1BFSm4aGjNhVcbnaqnfH42FhhS7pKCN/view?usp=share_link

**Reply by tuangd** [reply]
This solved the problem. I ran the first two lines and install the xformer

**Reply by TheLastBen** [reply]
Great! I'll add the libs, thanks

**Reply by Aribk7** [reply]
yooo https://twitter.com/aribk24/status/1601672936582103041?s=46&t=j9K4mM4gaD2ZtiPX71GH3A


people are having problems using the collab can u plz check the comments
